### PR TITLE
add audit log

### PR DESCRIPTION
**Pull request by donatelloraphael**

### CODE REVIEW

File: backend/src/handlers/items/updateBlog.js [hunk 1]
```diff
@@ -60,7 +60,10 @@ module.exports = (app) => {
       logger.debug('Mongo connected');
 
       const query = { $set: { 'info.editorial': flag } };
-      await dbClient.db().collection('items').updateOne({ _id: blogId }, query);
+      await Promise.all([
+        dbClient.db().collection('items').updateOne({ _id: blogId }, query),
+        mongo.logAdminActions(dbClient, req.identity.userId, 'toggle editorial', blogId, { flag }),
```
Comment (literakl) [inline backend/src/handlers/items/updateBlog.js:65]: blog is managed by users not admins

**Review by literakl — APPROVED** [review]
please fix blog audit later

File: backend/src/handlers/items/updateBlog.js (hunk 1 above)
Comment (donatelloraphael) [reply backend/src/handlers/items/updateBlog.js:65]: Even updating the editorial status? Doesn't that currently require admin role?

**Comment by literakl** [comment]
I see. Github showed only partial code snippet

Dne so 27. 3. 2021 21:07 uživatel donatelloraphael ***@***.***>
napsal:

**Pull request merged**